### ISSUE TITLE
libraries.io dependents badge with scoped NPM packages does not work

### ISSUE THREAD

**Issue by johnwebbcole** [badges/shields#4025]
libraries.io dependents badge with scoped NPM packages does not work

I'm trying to add a badge using libraries.io dependents for a scoped package.  I noticed that [libraries.io encodes the scoped name](https://libraries.io/npm/@jwc%2Fjscad-utils) into the package name, like `@jwc%2Fjscad-utils`, but this doesn't work with shields.io.

```
![Dependents](http://img.shields.io/librariesio/dependents/npm/@jwc%2Fjscad-utils
```

![Dependents](http://img.shields.io/librariesio/dependents/npm/@jwc%2Fjscad-utils)

If there is already a way to use scoped NPM packages, I wasn't able to find it in the documentation or another issue pointing to more info.

Thanks!

**Comment by calebcartwright** [comment]
Thanks for reporting this @johnwebbcole! If anyone is interested in fixing this bug, the solution here will probably be very similar to the ones in #3274 and #3570

**Comment by paulmelnikow** [comment]
@johnwebbcole Any chance you'd be interested in taking a look at this?

**Comment by netsgnut** [comment]
Just stumbled upon this problem right now. Will try and see if I could hack something out of it.

**Comment by calebcartwright** [comment]
Closed via #4151
